spec/resources/scanned_resources/scanned_resources_controller_spec.rb

```diff
@@ -201,6 +201,21 @@
       end
     end
 
+    context "when an attribute has leading / trailig spaces" do
+      let(:resource) { FactoryBot.create_for_repository(:scanned_resource) }
+      let(:params) do
+        {
+          source_metadata_identifier: " CD-18264 "
```

Comment:
I'm 👍 on the implementation, but not sure about using an invalid source metadata identifier in this test.

Reply:
hm, good point. I was using the data from the error, but it's incidental that it was bad. I'll sub in something valid.